tsconfig.json

```diff
@@ -16,13 +16,13 @@
     "incremental": true,
     "plugins": [
       {
-        "name": "next",
-      },
+        "name": "next"
+      }
     ],
     "paths": {
-      "@/*": ["./*"],
-    },
+      "@/*": ["./*"]
+    }
   },
   "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
-  "exclude": ["node_modules"],
+  "exclude": ["node_modules"]
```

Comment:
this diff is because your editor isn't sync'd with our eslint settings, I think? i don't remember. maybe prettier is in our pipeline, i forget.

Reply:
got changed after running `npm run prettier`. 
using nvm 18.17.0